README.md

```diff
@@ -17,6 +17,12 @@ To download the tool you have to have Node.js installed on your machine. After y
 $ npm install -g speedo
 ```
 
+## Update to newest versionNumber
+
+```sh
+$ npm install -g speedo@latest
+```
+
```

Comment:
I would add a section to verify that the newest version is used with `speedo --version`.

Or even recommend:
```
$ speedo --version
// => 1.2.2
$ npm install -g speedo@latest
$ speedo --version
// => 1.2.3
```

Reply:
added